docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.cn.md

```diff
@@ -20,7 +20,7 @@ writeConfiguration ::=
   'WRITE' '(' ('WORKER_THREAD' '=' workerThreadPoolSize ',')? ('BATCH_SIZE' '=' batchSize ',')? ('SHARDING_SIZE' '=' shardingSize ',')? (rateLimiter)? ')'
 
 dataChannel ::=
-  'STREAM_CHANNEL' '(' 'TYPE' '(' 'NAME' '=' algorithmName ',' 'PROPERTIES' '(' propertyDefinition ')'
+  'STREAM_CHANNEL' '(' 'TYPE' '(' 'NAME' '=' algorithmName propertyDefinition
```

Comment:
`propertyDefinition` is removed at line 40, and comma missed after algorithmName.